tests/Js/ChangeEventTest.php

```diff
@@ -3,6 +3,7 @@
 namespace Behat\Mink\Tests\Driver\Js;
 
 use Behat\Mink\Tests\Driver\TestCase;
+use Behat\Mink\Tests\Driver\Util\FixturesKernel;
```

Comment:
This might not be needed anymore, because `WEB_FIXTURES_DIR` and another constant were moved into the `Behat\Mink\Tests\Driver\TestCase` class.